content/en/docs/access-control-and-account-management/external-authentication/use-an-ldap-service.md

```diff
@@ -83,19 +83,19 @@ This document describes how to use an LDAP service as an external identity provi
    
    {{</ notice >}}
    
-5. If you are using KubeSphere 3.2.1, run the following command after configuring LDAP and wait until `ks-installer` is up and running:
+6. If you are using KubeSphere 3.2.0, run the following command after configuring LDAP and wait until `ks-installer` is up and running:
 
    ```bash
    kubectl -n kubesphere-system set image deployment/ks-apiserver *=kubespheredev/ks-apiserver:release-3.2
```

Comment:
suggest to update the command:
kubectl -n kubesphere-system set image deployment/ks-apiserver *=kubesphere/ks-apiserver:v3.2.1

Reply:
[Updated](https://github.com/kubesphere/website/pull/2213/commits/1fd2298ae8e4d156e78a832c638c6a6d259ea077).